### DISCUSSION TITLE
是否支持cf部署？

### DISCUSSION THREAD

**Discussion by xingpingcn** [lobehub/lobehub#2656] [General Question] [answered]
Finished
22:38:01.350 | Note: No functions dir at /functions found. Skipping.
22:38:01.350 | Validating asset output directory
22:38:01.991 | Deploying your site to Cloudflare's global network...
22:38:04.682 |  
22:38:04.803 | ✘ [ERROR] Error: Pages only supports files up to 25 MiB in size
22:38:04.803 |  
22:38:04.804 | cache/webpack/client-production/0.pack is 117 MiB in size

这是不是意味着不支持cf部署了？因为vercel和netlify都有函数响应时长限制，超过10秒就断开，根本没法用，只能尝试用cf了。

**Comment by arvinxx** [comment, marked as answer]
目前因为 CF 不支持 nodejs runtime ，所以没法在 Cloudflare 上部署。 超过 10s 断开的问题你充个钱也能解决的吧… 要免费用自己部署成 Docker 也可以的

**Reply by xingpingcn** [reply]
日志显示的是用next.js build成功了，是最后部署的时候说单个文件过大，不给部署。

**Reply by arvinxx** [reply]
如果是体积大的问题，或许可以试试看 CF 的付费版？我记得好像付费版单个文件的尺寸能更大一些
